### PR TITLE
Update SwiftFormat to 0.48.11

### PR DESCRIPTION
**Pull request by slashmo**
Updated SwiftFormat to be consistent with [swift-distributed-tracing](https://github.com/apple/swift-distributed-tracing).

### CODE REVIEW

File: Sources/InstrumentationBaggage/Baggage.swift [hunk 1]
```diff
@@ -99,7 +99,7 @@ extension Baggage {
     /// in order to inform other developers that the lack of baggage passing was not done on purpose, but rather because either
     /// not being sure where to obtain a baggage from, or other framework limitations -- e.g. the outer framework not being
     /// baggage aware just yet.
-    public static var topLevel: Baggage {
+    static var topLevel: Baggage {
```
Comment (ktoso) [inline Sources/InstrumentationBaggage/Baggage.swift:102]: this is a bad rule in swift format, can we not do it?

**Review by ktoso — APPROVED** [review]
LGTM, but can we not do the public on extensions rule?

**Comment by slashmo** [comment]
> LGTM, but can we not do the public on extensions rule?
> @ktoso 

Yep, I'll disable the rule.

File: Sources/InstrumentationBaggage/Baggage.swift (hunk 1 above)
Comment (slashmo) [reply Sources/InstrumentationBaggage/Baggage.swift]: Agreed, I disabled it.

**Pull request merged**